doc/md/fundamentals/1-basic-syntax/8-functions.md

```diff
@@ -0,0 +1,44 @@
+---
+sidebar_position: 8
+hide_table_of_contents: true
+---
+
+# Functions
+
+Functions in Motoko can be take on a variety of different attributes, the most basic of which is if the function is public or private. Public functions are callable by users or other [canisters](https://internetcomputer.org/docs/building-apps/essentials/canisters), while private functions can only be called within the program that defines it.
```

Comment:
```suggestion
Functions in Motoko can take on a variety of different attributes, the most basic of which is if the function is public or private. Public functions are callable by users or other [canisters](https://internetcomputer.org/docs/building-apps/essentials/canisters), while private functions can only be called within the program that defines it.
```

Reply:
Technically, public and private isn't a property of function but of declarations within the body of a actor, module or object - but perhaps that is too much information in this context.